test/mpi/mpi_t/mpit_isendirecv.c

```diff
@@ -118,65 +117,67 @@ static int compare_vars(int rank, int target, int num_nics)
         if (rank < num_pairs) {
             if (expected_mapping[0].nic_sent_bytes_count[idx] != nic_sent_bytes_count[idx]) {
                 printf
-                    ("rank=%d --> target=%d: Actual sent byte count=%ld through NIC %d does not match with the "
-                     "expected sent byte count=%ld\n", rank, target, nic_sent_bytes_count[idx], idx,
-                     expected_mapping[0].nic_sent_bytes_count[idx]);
+                    ("rank=%d --> target=%d: Actual sent byte count=%lld through NIC %d does not match with the "
+                     "expected sent byte count=%lld\n", rank, target, nic_sent_bytes_count[idx],
+                     idx, expected_mapping[0].nic_sent_bytes_count[idx]);
```

Comment:
should these be `%llu`?

Reply:
Will fix.

Reply:
https://github.com/pmodels/mpich/pull/5067/commits/b83da72b27f8bc861d836843e8f1c1a15cfeb6c4